### ISSUE TITLE
"Can't send lightning payments yet"

### ISSUE THREAD

**Issue by woojoo666** [btcontract/wallet#94]
I deposited some bitcoin to my wallet and then tried to pay a lightning invoice, and I got the error "Can't send Lightning payments yet because first you need to receive at least 0 sat to make some space for sending". What does this mean? Does this wallet not support using bitcoin to send payments across the lightning network?

**Comment by btcontract** [comment]
Have you deposited funds as a chain transaction?  If yes that would explain it since SBW has two wallets inside: a regular chain wallet which can only do regular chain transactions and a Lightning wallet which can only do Lightning payments.

If you want to use SBW specifically as Lightning wallet then current options are:

1. Send a chain transaction into SBW (that is supposedly done already) and then use those funds to open an LN channel (for example by scanning some node QR at 1ml.com).
2. Or get an incoming channel from services like lnbig.com and then send an LN payment into wallet.
3. Or send an LN payment into default hosted channel present in a wallet, but be warned that it is a custodial type of channel (you can swipe hosted channel card and remove it if it's not your thing).

> Can't send Lightning payments yet because first you need to receive at least 0 sat to make some space for sending

What is happening here is that I assume you have a single default hosted channel in a wallet with a zero balance, so app is asking you to send an LN payment in to get some spendable LN balance.

A message is not too clear though and I think it's worth to improve it, thanks for reporting this.

**Comment by woojoo666** [comment]
yeah I just started reading about channels before you posted. I was just using the default channel. But thanks for all of this information, the app (and LN in general) is starting to make sense now.

To be honest all I wanted to do was try some web service that took accepted LN payments, I had no idea it was this complicated. It's not like other cryptos where I can just convert on an exchange or swap using a DEX. Not only do I have to choose a channel, it seems like I have to [choose multiple, and I can't just choose the most popular ones](https://www.reddit.com/r/lightningnetwork/comments/olf33f/how_do_i_choose_with_who_to_open_a_new_channel/). 90% of the "beginner" articles explain how lightning network works, and don't explain how to actually use it. https://1ml.com seems like the main resource people use to find channels but it reads like a stock market display with cryptic numbers and hashes everywhere. It makes me wonder what Lightning Network was designed for, since it's clearly not designed for the general public to just be able to pay each other.

But anyways sorry for my off-topic ramble. I think the error message could use some fixing up but other than that I think the issue was mainly my misunderstanding. Thanks for the help.
